app/src/main/java/com/example/wearit/ui/WardrobeScreen.kt

```diff
@@ -251,41 +251,68 @@ fun WardrobeNavigationSection(
 
 @Composable
 fun AddItemDialog(
-    onDismiss: () -> Unit,
+    closeDialog: () -> Unit,
     originalPhoto: Bitmap?,
     noBgPhoto: Bitmap?,
     saveItem: (bitmap: Bitmap) -> Unit
 ) {
-    Log.d("WardrobeScreen", (originalPhoto != null).toString() + " " + (noBgPhoto != null).toString())
-    
     var showOriginal by remember { mutableStateOf(true) }
     
     Dialog(
-        onDismissRequest = onDismiss
+        onDismissRequest = closeDialog
     ) {
         Card(
-            modifier = Modifier.fillMaxWidth(0.95f)
+            modifier = Modifier
+                .fillMaxWidth(0.95f)
         ) {
             Column(
                 modifier = Modifier.fillMaxWidth()
             ) {
                 if (showOriginal && originalPhoto != null) {
-                    Image(bitmap = originalPhoto.asImageBitmap(), contentDescription = null)
+                    Image(
+                        bitmap = originalPhoto.asImageBitmap(),
+                        contentDescription = null,
+                        modifier = Modifier
+                            .padding(20.dp)
+                            .fillMaxWidth(),
+                        contentScale = ContentScale.FillWidth
+                    )
                 }
                 else if (!showOriginal && noBgPhoto != null) {
-                    Image(bitmap = noBgPhoto.asImageBitmap(), contentDescription = null)
+                    Image(
+                        bitmap = noBgPhoto.asImageBitmap(),
+                        contentDescription = null,
+                        modifier = Modifier
+                            .padding(20.dp)
+                            .fillMaxWidth(),
+                        contentScale = ContentScale.FillWidth
+                    )
                 }
-                Row() {
-                    Button(onClick = { onDismiss }) {
+
+                Row(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(20.dp, 0.dp, 20.dp, 10.dp),
+                    horizontalArrangement = Arrangement.SpaceBetween
+                ) {
+                    Button(onClick = closeDialog) {
```

Comment:
jeśli się nie mylę ten przycisk "cancel" jest zbędny, bo okno dialogowe zamyka się samo po kliknięciu w dowolnym miejscu poza jego obszarem (i to też jest intuicyjny sposób na opuszczenie tego okna jeśli chodzi o użytkownika). ewentualnie pasowałby jakiś przycisk X w prawym górnym roku tego okienka.
![Screenshot_6](https://user-images.githubusercontent.com/64137088/205692521-ead4dd8c-4a22-402f-ba27-c912d62d6a1c.png)